src/pages/spawn/spawnHost/SpawnHostActionButton.tsx

```diff
@@ -93,7 +93,7 @@ export const SpawnHostActionButton: React.FC<{ host: MyHost }> = ({ host }) => {
       {action ? (
         <PaddedButton
           disabled={loading}
-          glyph={<Icon glyph={glyph} />}
+          glyph={<Icon fill="black" glyph={glyph} />}
```

Comment:
It looks like button icons in LG actually [use gray.dark2](https://github.com/mongodb/leafygreen-ui/blob/4afff2bcbbecbc26468f49111602bca40c91353b/packages/button/src/ButtonIcon.tsx#L53)!

Reply:
Nice catch!